src/docsView.ts

```diff
@@ -133,7 +133,7 @@ export class DocsViewViewProvider implements vscode.WebviewViewProvider {
 				
 				<title>Documentation View</title>
 			</head>
-			<body>
+			<body style = "${this.getStyleOverrides()}">
```

Comment:
Instead of using a style attribute, how about using a css variable for the font-size and having the script simple update this variable? This prevents the possibility of injecting html into the webview

Here's an example of that from VS Code's webview code: 			
		https://github.com/microsoft/vscode/blob/bc0645202f34edf82aa317f8915dd2b4d1b249ad/src/vs/workbench/contrib/webview/browser/pre/main.js#L229